mainfile

```diff
@@ -10,35 +10,35 @@
     DC MOTORS AND AN ULTRASONIC SENSOR. 
 */
 
-
```

Comment:
**i think we might need a method to determine how long it takes to turn a certain number of degrees so turnTime is the method with a parameter of degrees and it will calculate how many seconds per degree or something like that. we can also just test this by having it turn 360 degrees, time it, divide by 360 and store that value in 'double turn'**
double turn = xxxx;
double turnTime(int degrees){
     return turn * degrees;
}
void moveForward(double distance){
for (int i = 0; i < calculateDistance(distance); i++){
     analogWrite(motor1,2,3,4, normal);
     ...
}
}
void turnRight(int degrees){
for (int i = 0; i < turnTime(degrees); i++){
analogWrite(motor 1,3, normal);
analogWrite(motor2,4, slow);
}
}
void turnLeft(int degrees){
for (int i = 0; i < turnTime(degrees); i++){
analogWrite(motor 1,3, slow);
analogWrite(motor2,4, normal);
}
}
**do we want to manually call each method for each segment of the route? i cant really think of another way to do it** 
moveForward(time);
moveRight(degrees);
moveLeft(degrees);
etc.... according to the sketch